src/python/pants/backend/go/lint/vet/rules_integration_test.py

```diff
@@ -142,7 +142,8 @@ def test_failing(rule_runner: RuleRunner) -> None:
     assert len(lint_results) == 1
     assert lint_results[0].exit_code == 2
     assert (
-        "./f.go:4:5: Printf format %s reads arg #1, but call has 0 args" in lint_results[0].stderr
+        "./f.go:4:5: fmt.Printf format %s reads arg #1, but call has 0 args"
```

Comment:
Does this bring other OSes though? Presumably CI had it as `Printf` for a reason. Maybe check if either or is there?

Reply:
FWIW I've also seen this fail on my Intel-based mac, but I never dug into what was going wrong